src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/WageTypeValidator.cs

```diff
@@ -151,6 +162,15 @@ private void ConfigureWageTypeValidator()
                 .WithErrorCode(ErrorCodes.CreateApprenticeship.WageTypeReason);
         }
 
+        private bool BeMonetaryValue(decimal? monetaryValue)
```

Comment:
Can this be moved to extensions?

Reply:
I'll sort this one in another PR. (db change already merged with breaking changes so need to get this onto master soon).